mkdocs_page_pdf/plugin.py

```diff
@@ -57,7 +57,7 @@ async def page_to_pdf (self, output_content, outputpath, filename):
         print('Page to pdf ' + os.path.join(outputpath, filename))
 
     def on_config(self, config):
-        onServe = config['dev_addr'].host+':'+str(config['dev_addr'].port) in config['site_url']
+        onServe = config['dev_addr'].host+':'+str(config['dev_addr'].port) in ( config['site_url'] or 'http://127.0.0.1:8000')
```

Comment:
This will make `mkdocs build` interpreted as serve if no `site_url`is defined so instead I would do this : 
```python
onServe = config['site_url'] is not None and config['dev_addr'].host+':'+str(config['dev_addr'].port) in config['site_url']
```